.devcontainer/devcontainer.json

```diff
@@ -5,11 +5,13 @@
 	"service": "node",
 	"workspaceFolder": "/project",
 	"settings": {
-		"terminal.integrated.shell.linux": "/bin/bash",
+		"terminal.integrated.defaultProfile.linux": "bash",
```

Comment:
`terminal.integrated.defaultProfile.linux` and `terminal.integrated.defaultProfile.linux` are two different things. I think we need both